### DISCUSSION TITLE
Created a github project based on Grott for Huawei inverter data relay

### DISCUSSION THREAD

**Discussion by JasperE84** [johanmeijer/grott#125] [Show and tell]
Hi Johan,

Thought you might like to check out this project I created on GitHub: [PyFusionSolarDataRelay](https://github.com/JasperE84/PyFusionSolarDataRelay)

It's largely based on the Grott structure and code. I refactored quite some bits though. Primarily to apply the [single responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle), implement a logging library, and change up the way exceptions are handled.

Maybe you can take some inspiration/example from it. But note that this is also my first real Python project. Do have some experience in other languages though.

Thanks for all the work in Grott, thanks to your work now also Huawei inverter users can fetch some data into influxdb and pvoutput :)

**Comment by johanmeijer** [comment]
Nice! I am going to look at it and see if I can use parts of it for Grott in future.
